src/app/clusters/group-key-mgmt-server/group-key-mgmt-server.cpp

```diff
@@ -401,7 +401,32 @@ ValidateKeySetWriteArguments(const chip::app::Clusters::GroupKeyManagement::Comm
     return Status::Success;
 }
 
-constexpr uint16_t GroupKeyManagementAttributeAccess::kClusterRevision;
+bool ValidateAndGetProviderAndFabric(chip::app::CommandHandler * commandObj, const chip::app::ConcreteCommandPath & commandPath,
```

Comment:
remove `validateand`. Maybe GetValidProviderAndFabric and even that seems a bit inconvenient because of the and.

Reply:
Yes. Too many Ands in the function name. I would suggest just GetProviderAndFabric(). Getting valid ones should be implied and checks done internally.

Reply:
Updated the code.